public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -72,6 +82,11 @@ export const LayerControlPanel = memo(
     mapState,
     zoom,
     mapConfig,
+    inDashboardMode,
+    timeRange,
+    refreshConfig,
+    filters,
+    query,
   }: Props) => {
```

Comment:
If we don't need layer control panel, why are we passing here? Why not move this to new component and update there? Layer Control Panel should not be involved if there is no panel to render.

Reply:
Just moved to parent component, because the props are used in useEffect, it's hard to move out to a new component